packages/rancher-monitoring/rancher-kube-state-metrics/generated-changes/patch/values.yaml.patch

```diff
@@ -10,37 +10,27 @@
 -  tag: ""
 +  registry: docker.io
 +  repository: rancher/mirrored-kube-state-metrics-kube-state-metrics
-+  tag: v2.10.1
++  tag: v2.12.0
    sha: ""
    pullPolicy: IfNotPresent
  
-@@ -12,6 +11,11 @@
+@@ -12,6 +11,9 @@
  # - name: "image-pull-secret"
  
  global:
 +  cattle:
-+    psp:
-+      enabled: false
 +    systemDefaultRegistry: ""
 +
    # To help compatibility with other charts which use global.imagePullSecrets.
    # Allow either an array of {name: pullSecret} maps (k8s-style), or an array of strings (more common helm-style).
    # global:
-@@ -97,8 +101,7 @@
+@@ -104,8 +106,7 @@
  kubeRBACProxy:
    enabled: false
    image:
 -    registry: quay.io
 -    repository: brancz/kube-rbac-proxy
 +    repository: rancher/mirrored-kube-rbac-proxy
-     tag: v0.14.0
+     tag: v0.18.0
```

Comment:
Actually looking into this, I do make this change later, but since you are looking at an individual commit its hard to tell. I'm going to mark this resolved, feel free to re-open